### PR TITLE
feat(analytics): Adding hooks for completing onboarding

### PR DESCRIPTION
**Pull request by ehfeng**

### CODE REVIEW

File: src/sentry/static/sentry/app/views/onboarding/configure/index.jsx [hunk 1]
```diff
@@ -83,6 +84,9 @@ const Configure = React.createClass({
 
   submit() {
     this.redirectUrl();
+    HookStore.get('analytics:onboarding-complete').forEach(cb => {
+      cb();
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/onboarding/configure/index.jsx]: nit but you don't need the {}, can do forEach(cb=>cb())

**Review by MaxBittker — APPROVED** [review]
lgtm 👍

**Comment by MaxBittker** [comment]
this doesn't pass tests though

**Comment by ehfeng** [comment]
Working on it. Having to re-setup all my dev environments on my personal machine.

**Pull request merged**